src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -319,6 +319,34 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # Default 0-sized value means unlimited rate.
 #tx_rate_limiter_max_rate = 0
 
+# pvpanic device is a simulated ISA device, through which a guest panic
+# event is sent to qemu, and a QMP GUEST_PANICKED event is generated.
+# This allows management apps to be notified and respond to the event.
```

Comment:
You could potentially add a little more detail here:

```toml
# Note that this can also detect if the init daemon has died. The init daemon is systemd if an image is specified
# with 'image=', or the Kata agent if an initrd if 'initrd=' is specified instead.
````